packages/block-library/src/post-content/block.json

```diff
@@ -2,7 +2,7 @@
 	"$schema": "https://schemas.wp.org/trunk/block.json",
 	"apiVersion": 3,
 	"name": "core/post-content",
-	"title": "Post Content",
+	"title": "Content",
```

Comment:
I know I'm thinking about it too much as a programmer. But just content seems to vague to me. 

The post content has become such an abstracted term for the content field.

Reply:
> The post content has become such an abstracted term for the content field.

For developers yes, but does "Post content" work when you're editing the images on a portfolio item?

Reply:
It is even strange to call it post content when you are building a *page* template

Reply:
If we kept the prefix it'd need to be something generic that works in any context... "Entry Content"? I'm not sure that makes it any less vague though. On balance it seems okay to try "Content" – we can always circle back if it proves troublesome.